migrations/3_deploy_mocks.js

```diff
@@ -1,5 +1,6 @@
 const assert = require('assert');
 const DeployerApp = require('./utils/DeployerApp');
+const AdminUpgradeabilityProxy = artifacts.require("./base/UpgradeableProxy.sol");
```

Comment:
We could add a message logging Admin's address

Reply:
You mean which account is the `admin` for the upgradeable proxies?

Reply:
I thought we would have one ProxyAdmin contract to reuse for all the contracts. As it's one per upgradeable contract I don't think we need to log every admin for every upgradeable contract.